packages/schematics/angular/ng-new/index.ts

```diff
@@ -62,13 +59,9 @@ export default function(options: NgNewOptions): Rule {
   };
 
   return chain([
-    mergeWith(
-      apply(empty(), [
-        schematic('workspace', workspaceOptions),
-        options.createApplication ? schematic('application', applicationOptions) : noop,
-        move(options.directory),
-      ]),
-    ),
+    schematic('workspace', workspaceOptions),
+    options.createApplication ? schematic('application', applicationOptions) : noop,
+    move(options.directory),
```

Comment:
Could also potentially use applyToSubTree and avoid the move.

Reply:
Isn't be `path` required to exists to use the `applyToSubTree`?